br/pkg/storage/gcs.go

```diff
@@ -88,6 +88,11 @@ type gcsStorage struct {
 	bucket *storage.BucketHandle
 }
 
+// DeleteFile delete the file in storage
+func (s *gcsStorage) DeleteFile(ctx context.Context, name string) error {
+	return errors.New("not implemented")
```

Comment:
... just implement this? you just call [`(*ObjectHandle).Delete`](https://pkg.go.dev/cloud.google.com/go/storage#ObjectHandle.Delete).